src_py/apiServer/experiment_flow_test.py

```diff
@@ -57,13 +58,13 @@ def print_test(in_str : str):
 if stderr: 
     print_test(stderr)
 else:
-    print_test(stdout)
+    print_test(stdout, False)
 stdout, stderr, rc = nerlnet_stop_cmd.sync(NERLNET_RUNNING_TIMEOUT_SEC)
 print_test(f'rc stop: {rc}')
 if stderr: 
     print_test(stderr)
```

Comment:
@GuyPerets106  @NoaShapira8  @ohad123 
After print_test of stderr add
exit(rc)